src/app/login/page.tsx

```diff
@@ -25,27 +23,23 @@ import {
   Lock,
 } from "@mui/icons-material";
 import { ThemeProvider, useTheme, Theme } from "@mui/material/styles";
-import { url } from "inspector";
-
 import { UserAPI } from "@/apis/UserAPI";
+import { toast, ToastContainer } from "react-toastify";
```

Comment:
Toast component is already created, why using react-toastify

Reply:
done